srml/sudo/src/lib.rs

```diff
@@ -14,8 +14,112 @@
 // You should have received a copy of the GNU General Public License
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
 
-//! The Example: A simple example of a runtime module demonstrating
-//! concepts, APIs and structures common to most runtime modules.
+//! # Sudo Module
+//!
+//! ## Overview
+//! 
+//! The Sudo module allows for a single account (called the "sudo key")
+//! to execute dispatchable functions that require a `Root` call
+//! or designate a new account to replace them as the sudo key.
+//! Only one account can be the sudo key at a time.
+//! 
+//! You can start using the Sudo module by implementing the sudo [`Trait`].
+//! 
+//! Supported dispatchables functions are documented as part of the [`Call`] enum.
+//! 
+//! ## Interface
+//! 
+//! ### Dispatchable
+//! 
+//! Only the sudo key can call the dispatchable functions from the Sudo module.
+//! 
+//! * `sudo` - Make a `Root` call to a dispatchable function.
+//! * `set_key` - Assign a new account to be the key of the superuser.
```

Comment:
should we add a line break between these, similar to the timestamp docs

Reply:
I thought standard markdown was not to have a whitespace line between list items?

Reply:
It depends on how users prefer it to appear in the browser. Example:

**Without** whitespace between list items:

<img width="794" alt="Screen Shot 2019-03-20 at 12 15 40 pm" src="https://user-images.githubusercontent.com/6226175/54680506-5471a180-4b0a-11e9-8616-9b210d69f55d.png">

**With** whitespace between list items:

<img width="857" alt="Screen Shot 2019-03-20 at 12 16 53 pm" src="https://user-images.githubusercontent.com/6226175/54680526-60f5fa00-4b0a-11e9-9629-8cedaf2d08f6.png">